app/controllers/users_controller.rb

```diff
@@ -1,7 +1,66 @@
 class UsersController < ApplicationController
 
-  def list
+  def show
     @users = User.all
   end
-  alias :index :list
+  alias :index :show
```

Comment:
You seem to be breaking REST conventions here by making
``` GET /users```
and
```GET /users/:id```
return the same thing.  REST conventions would make
index => list of all users
show => single user by params[:id]